win32ss/user/winsrv/concfg/font.c

```diff
@@ -40,16 +40,20 @@ CodePageToCharSet(
 }
 
 HFONT
-CreateConsoleFontEx(
+CreateConsoleFontExEx(
```

Comment:
Suggestion to call the function `CreateConsoleFontInternal` (instead of ExEx) 😂

Reply:
OK.